examples/flutter_view/lib/main.dart

```diff
@@ -0,0 +1,85 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+final String _CHANNEL = "increment";
+final String _EMPTY_MESSAGE = "";
```

Comment:
Dart constants should be lowerCamelCase, see https://www.dartlang.org/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names.

You should probably declare them `const`, and since they are used only by `MyHomePageState`, they could be declared `static const` in there.